resources/qml/ViewOrientationControls.qml

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2018 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.2
+import QtQuick.Controls 1.1
+import QtQuick.Controls.Styles 1.1
+
+import UM 1.4 as UM
+
+// View orientation Item
+Row
+{
+    id: viewOrientationControl
+
+    spacing: 2 * screenScaleFactor
```

Comment:
Use the margins in the theme for this.

Reply:
Done in 17b77c58eff28e52f9d070cbc5a80c3dd5fcc4ec